src/components/Form/Form.js

```diff
@@ -97,6 +101,15 @@ export function Form({ children, config }) {
     // This just makes the rest of this easier to read
     const fieldConfig = config.fields[fieldName]
 
+    const doValidation = (field) => {
+      return fieldConfig.validators
```

Comment:
It's crazy to me that this same exact code is being done in the `analyticsWrapper` and we're not just passing in the analytics event we want called and reconciling here :/